pyproject.toml

```diff
@@ -286,7 +286,6 @@ max-complexity = 21
 "nilearn/glm/tests/test_thresholding.py" = ["D103"]
 "nilearn/glm/tests/test_utils.py" = ["D103"]
 "nilearn/image/**/{tests}/*" = ["D103"]  # 75 errors to fix
-"nilearn/maskers/*" = ["B028"]  # 17 errors to fix
```

Comment:
will first tackle the modules that are listed with a B028 error here
then will apply to the rest of the codebase